0000-template.md

```diff
@@ -1,3 +1,14 @@
+<!---
+Copyright <fill in year> The Rust Project Developers. See the COPYRIGHT
```

Comment:
@est31 That would be entirely fine with me. I'd just like to get this sorted out before we further propagate it.